@here/harp-lines/lib/Lines.ts

```diff
@@ -34,21 +34,19 @@ export function createLineGeometry(
     const V = geometry.vertices;
     const I = geometry.indices;
 
-    const N: number[] = [];
-    const L: number[] = [];
-    const tangents: Array<[number, number]> = [];
-    const baseVertex = geometry.vertices.length / 12;
+    const L = new Array<number>(pointCount);
+    const tangents = new Array<[number, number]>(pointCount - 1);
+    const baseVertex = V.length / (highPrecision ? 14 : 12);
```

Comment:
Was this a bug? Why do high precision points start at 14?

Reply:
Stride size for high precision lines is 14, for regular ones is 12. They cannot be mixed within one geometry so this does look like a bug to me.

Reply:
Ok, that makes sense. 
Can you please either add a:
"TODO: take away magic numbers"
Or add some const numbers to the top of this file? I notice that the `static createGeometry(` function also relies on these numbers.

Reply:
Added constants instead of these magic numbers.